lib/database.py

```diff
@@ -30,7 +30,7 @@ def get_download_count(self, dataset_id=None):
             return
 
         cursor = self.conn.cursor()
-        cursor.execute("SELECT COUNT(id) FROM guestbookresponse WHERE downloadtype = 'Download' AND dataset_id = %s;", [str(dataset_id)])
+        cursor.execute("SELECT COUNT(id) FROM guestbookresponse WHERE dataset_id = %s;", [str(dataset_id)])
```

Comment:
I'm curious why the downloadtype clause was removed? Without it the query will count TwoRavens responses in addition to downloads.

Reply:
I see now. There's a new filedownload table. This query will need to change to a JOIN. Something like `SELECT COUNT(g.id) FROM guestbookresponse g LEFT JOIN filedownload f on g.id = f.guestbookresponse_id WHERE g.dataset_id = %s;`.

That said, this count is now available via the Make Data Count endpoints in the API, assuming they're enabled. At some point, I would like to get rid of the database class entirely and just use the API.